app/templates/documents.rb

```diff
@@ -2,8 +2,8 @@
 
 class Documents
   include Templatable
-  LIGHT_STEMMERS = { fr: "french", de: "german", es: "spanish", it: "italian", pt: "portuguese" }
-  STANDARD_STEMMERS = { en: "english", fi: "finnish", hu: "hungarian", ru: "russian", sv: "swedish" }
+  LIGHT_STEMMERS = { fr: 'french', de: 'german', es: 'spanish', it: 'italian', pt: 'portuguese' }
```

Comment:
Rubocop's rule, "Prefer single-quoted strings when you don't need string interpolation or special symbols."